Friendly-plans/app/src/main/java/database/entities/PlanTemplate.java

```diff
@@ -62,6 +70,18 @@ public void setName(String name) {
         this.name = name;
     }
 
+    public void setTasksWithThisPlan(TaskTemplate task) {
```

Comment:
I think this code should be in PlanTemplateRepository, we should avoid modifying generated classes.